cmd/edgectl/aes_login.go

```diff
@@ -86,7 +87,7 @@ func do_login(kubeinfo *k8s.KubeInfo, context, namespace, hostname string, openI
 	}
 
 	// Output
-	url := fmt.Sprintf("https://%s/edge_stack/admin/#%s", hostname, tokenString)
+	url := fmt.Sprintf("https://%s/edge_stack/admin/?first-install=%v#%s", hostname, firstInstall, tokenString)
```

Comment:
I'm thinking maybe we want the `?first-install=%v` string to be appended ONLY if it's a firstInstall, to prevent "hackers" from tempering with the URL as it'll show up in their browser as `?first-install=false`

Reply:
interesting, I actually had the same thought.  But of course they can always type it in themselves after they saw it once.